tests/test.py

```diff
@@ -1540,6 +1597,77 @@ def test_only_alpha_client_id(self):
         self.assertEqual(create_client_keypair(POISON_KEY_PATH), 1)
 
 
+class TestAcraConfigUIGenAuth(unittest.TestCase):
+    def testUIGenAuth(self):
+        self.assertEqual(manage_basic_auth_user('set', 'test', 'test'), 0)
+        self.assertEqual(manage_basic_auth_user('set', CONFIG_UI_BASIC_AUTH['user'], CONFIG_UI_BASIC_AUTH['password']), 0)
+        self.assertEqual(manage_basic_auth_user('remove', 'test', 'test'), 0)
+        self.assertEqual(manage_basic_auth_user('remove', 'test_unknown', 'test_unknown'), 1)
+
+
+class TestAcraConfigUIWeb(BaseTestCase):
+    def setUp(self):
+        try:
+            self.proxy_1 = self.fork_proxy(
+                self.PROXY_PORT_1, self.ACRA_PORT, 'keypair1', zone_mode=True, commands_port=self.PROXY_COMMAND_PORT_1)
+            self.acra = self.fork_acra(zonemode='true', enable_http_api='true')
+            self.configui = self.fork_configui(proxy_port=self.PROXY_COMMAND_PORT_1, http_port=self.CONFIG_UI_HTTP_PORT)
+        except Exception:
+            self.tearDown()
+            raise
+
+    def tearDown(self):
+        try:
+            os.unlink('configs/acraserver.yaml')
+        except Exception as e:
+            print(e)
+        stop_process([self.configui])
+        args = ['killall', '--signal=SIGTERM', 'acraserver']
+        try:
+            subprocess.call(args, cwd=os.getcwd(), timeout=PROCESS_CALL_TIMEOUT)
+        except Exception as e:
+            print(e)
```

Comment:
on error send sigkill signal to stop process at all because sigterm may not finished and may be freezed

Reply:
done